tools/rna_tools/pipmir/pipmir.xml

```diff
@@ -0,0 +1,230 @@
+<tool id="pipmir" name="PIPmiR PIPELINE" version="0.1.0">
+
+    <description>a method to identify novel plant miRNA</description>
+
+    <requirements>
+        <!-- conda dependency -->
+        <requirement type="package" version="1.1">pipmir</requirement>
+    </requirements>
+
+    <command>
+<![CDATA[
+
+    samtools sort
+
+    ## output in BAM format
+    -O BAM
+
+    ## output file name
+    -o test_sorted.bam
+
+    "$input_bam"
```

Comment:
but it was not practiced here,  https://github.com/galaxyproject/tools-iuc/blob/596a23abc0c657eff218d16df685b427ae7cdd0a/tools/extract_genomic_dna/extract_genomic_dna.xml#L15

but changed to signle quote any way.